blackbox/docs/sql/statements/create-table.rst

```diff
@@ -224,7 +224,7 @@ Several restrictions apply to columns that can be used here:
 .. _with_clause:
 
 ``WITH``
---------
+========
```

Comment:
Afaik the changes in this file are unrelated to this PR, I'd prefer having them in a dedicated PR or at least commit.

Reply:
valid point